### ISSUE TITLE
404 - Page not found

### ISSUE THREAD

**Issue by InfiniteMort** [sharpdx/sharpdx.github.io#1]
404 - File Not Found

Whoops! Could not find that page.
Hi there, SharpDX website has been recently migrated to a new platform and have broken many links.

If you don't mind, please log an issue with the URL you were trying to reach. Thanks!
-------
http://sharpdx.org/documentation/api/t-sharpdx-multimedia-soundstream

**Comment by StijnBrouwer** [comment]
@xoofx, could you at least provide an archived version of the old documentation? SharpDX is nearly impossible to use without those docs.

**Comment by Swizzy** [comment]
Indeed, even so, the old stuff is useful for those using the older versions, not everyone will be using the new 3.0 version for various reasons

**Comment by xoofx** [comment]
@StijnBrouwer I can't, as it is requiring hosting this with Joomla+mysql on a linux box and I don't want to manage this any more.

The API documentation will be ported in some ways to github. I haven't yet decided exactly how I will do it.

I was expecting to generate only pages between the SharpDX managed names, the native name and a link to the msdn DirectX API. Do you need more than this? (more than 90% of the API is actually the same than the native API, so duplicating the API is not really helpful)

**Comment by InfiniteMort** [comment]
The web address was one being reported by Google, when searching for a data type from XAudio2. If there is a newer documentation page somewhere, I assume Google will index that instead, shortly.

**Comment by xoofx** [comment]
Hi,
I have updated the wiki now hosted on http://sharpdx.org/wiki (as I was unable to get a proper wiki on github).

This wiki contains a rudimentary Class Library API reference that contains links to the original msdn documentation. It doesn't contain documentation about pure managed types added by SharpDX (e.g: types from `SharpDX.Mathematics`)

In the future, I will evaluate to build back a full documentation from the managed API, maybe by using [docfx](http://dotnet.github.io/docfx/)

**Comment by InfiniteMort** [comment]
I had a look at the new http://sharpdx.org/wiki page, in order to find some information about which SharpDX mapped to the DirectX calls I was looking for. 

I didn't find what I was searching for, but I believe I will be able to manage without it, at least for now.

Just to fill you in on what I looked for on the site, this is what I did:
I was reading an MSDN article on how to draw text, using D2D and DirectDraw (https://msdn.microsoft.com/en-us/library/windows/desktop/dd756692(v=vs.85).aspx).

The article described needing "D2D1CreateFactory" to create a factory, so I searched for that, in the Direct2D1 API reference (http://sharpdx.org/wiki/class-library-api/direct2d1.html). Not finding that result, I searched for "CreateFactory", without luck and finally searched for "Factory". I got several hits on "Factory", but none that explained how to create an instance of the factory. 

Although I wasn't able to find the reference to "D2D1CreateFactory" in the wiki, I did guess that a Factory class probably existed and could be created from within the "SharpDX.Direct2D1" namespace. I looked in the source code and searching for "D2D1CreateFactory" led me to the DllImport, which in turn led me to the CreateFactory and finally the constructor of the "SharpDX.Direct2D1.Factory" class.

I wasn't expecting the rudimentary class library API to contain a lot of information yet, but knowing my thought process, when reading the wiki, might be useful for any future work that may be spent on it.
